src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/RuntimeBasicWriter.cs

```diff
@@ -182,8 +182,23 @@ public override void WriteHtmlAttributeValue(CSharpRenderingContext context, Htm
                 .WriteStringLiteral(node.Prefix)
                 .WriteParameterSeparator()
                 .Write(prefixLocation.ToString(CultureInfo.InvariantCulture))
-                .WriteParameterSeparator()
-                .WriteStringLiteral(node.Content)
+                .WriteParameterSeparator();
+
+            // Write content
+            for (var i = 0; i < node.Children.Count; i++)
+            {
+                if (node.Children[i] is RazorIRToken token && token.IsHtml)
```

Comment:
Thanks GitHub, the diff didn't expand the way I thought and it looked like this was under the `HtmlAttributeIRNode`